src/services/message-service.ts

```diff
@@ -30,6 +43,30 @@ export const createGetQueueMessage = (): ISocketMessage => {
   return createMessage({}, QUEUE_LIST);
 };
 
+export const createPingMessage = (): ISocketMessage => {
+  return createMessage({}, PING);
+};
+
+export const createGetAvailableQueueMessage = (): ISocketMessage => {
+  return createMessage({}, AVAILABLE_CHAT)
+}
+
+
+export const createJoinChatMessage = (
```

Comment:
Fire args i en funksjon er litt mye, lag heller en med builder pattern som gjort under 🛩

Reply:
Eller send inn et object og deconstructer. Så kan du lage et interface for det objeket 👍